launch-game.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+TEMPLATE_LAUNCHER=$(python -c "import os; print(os.path.realpath('$0'))")
+TEMPLATE_ROOT=$(dirname "${TEMPLATE_LAUNCHER}")
+source ${TEMPLATE_ROOT}/mod.config
+
+MOD_SEARCH_PATHS="./../.."

```

Comment:
It depends.  Git doesn't track files, per se, but blobs of text.  The history will be retained if, in a single commit, only a rename happens.  However, if you rename and change code in the same commit then it depends on how much you change.  In general renaming files and folders should be kept as its own commit.